### ISSUE TITLE
Finetuning loss of Qwen3vl-moe-30B drops to zero suddenly

### ISSUE THREAD

**Issue by yanwj21** [QwenLM/Qwen3-VL#1552]
When finetuning Qwen3vl-moe-30B, the loss drops to zero after several steps.

**Comment by JJJYmmm** [comment]
I think it's related to multinode fine tuning. cc @John-Ge .

**Comment by yanwj21** [comment]
> I think it's related to multinode fine tuning. cc [@John-Ge](https://github.com/John-Ge) .

When I change my GPU from PPU to A100 (environment may also changed), the problem seems solved, but I'm still unsure if the problem is related to device or environment

**Comment by JJJYmmm** [comment]
We’ve encountered this issue before in a multinode setup. If it happens again, please feel free to report it—we’ll investigate thoroughly.

**Comment by yanwj21** [comment]
> We’ve encountered this issue before in a multinode setup. If it happens again, please feel free to report it—we’ll investigate thoroughly.

Forgot to mention that I didn't adopt a multi-node setup, I just used a single node with 8 gpu.

**Comment by zhushiding** [comment]
I also encountered this issue. When I trained Qwen3vl-moe-30B on multiple machines and multiple GPUs, the loss immediately dropped to 0.

<img width="1203" height="296" alt="Image" src="https://github.com/user-attachments/assets/e32de1a6-e20f-4a4c-9f8e-ba5dd13cde28" />

**Comment by DawnHH** [comment]
你好，我也遇到了这个问题，在训练的第二步就出现了，我是用8机64卡H20训练的，为了不爆显存设置的batch_size=1
grad_accum_steps=1，--per_device_eval_batch_size $((batch_size * 1))，其他设置与sft_30a3b.sh脚本一致，使用的训练数据sft qwen2.5vl32b是没问题的：

<img width="2763" height="355" alt="Image" src="https://github.com/user-attachments/assets/c93ea5a3-a00c-4013-be0c-d89f9bb2e004" />

**Comment by LHB-kk** [comment]
same issue

**Comment by BiEchi** [comment]
Same issue! However, we're only seeing this issue on a single node of B200x8. It does not happen when in multi-node setting (2-node or 3-node). 

cc @JJJYmmm
